src/components/Books.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { removeBook } from '../redux/books/books';
+
+const Book = ({ title, author, id }) => {
+  const dispatch = useDispatch();
+
+  const handleRemove = () => {
+    dispatch(removeBook(id));
+  };
+
+  return (
+
+    <div className="book">
+
+      <h2 className="title">{title}</h2>
+      <h6 className="author">{author}</h6>
+
+      <button className="btn-remove" type="button" onClick={handleRemove}>
```

Comment:
Removing a book is not working as expected. Kindly have a look at [this video](https://www.loom.com/share/157ac51d2b664b7cb0dd583bd7006480) I recorded where I demonstrated what exactly is going wrong. Kindly fix this issue and then submit for a re-review.

Reply:
@muneebulrehman thanks my bro you are the only reviewer who recorded a video and helped me a lot.

Reply:
Great job @muneebulrehman for the video👏🏻 🥇